src/main/resources/xslt/transformations/ddi2odt/drivers.fods

```diff
@@ -1086,6 +1086,20 @@
       <text:p>A GenerationInstruction activates the CalculatedVariable driver.</text:p>
      </table:table-cell>
     </table:table-row>
+    <table:table-row table:style-name="ro1">
+     <table:table-cell table:number-columns-repeated="3"/>
+    </table:table-row>
+    <table:table-row table:style-name="ro1">
+     <table:table-cell office:value-type="string" calcext:value-type="string">
```

Comment:
Is this the implementation driven by the changes on the master branch ?
We have a in the roadmap the Control driver expected, is this the same implemenation ?